src/ImageSharp/ImageFrame{TPixel}.cs

```diff
@@ -267,6 +301,23 @@ internal ImageFrame<TPixel2> CloneAs<TPixel2>()
             return target;
         }
 
+        /// <summary>
+        /// Clears the bitmap.
+        /// </summary>
+        /// <param name="parallelOptions">The parallel options.</param>
+        /// <param name="value">The value to initialize the bitmap with.</param>
+        public void Clear(ParallelOptions parallelOptions, TPixel value) {
```

Comment:
Clearing is a process and belongs in the processing namespace; we don't want to clutter the Image class. We also never want to expose the ParallelOptions as a field. They are part of the Configuration Class and are globally set.

Reply:
Changed method from public to internal. I'd just see this as initialization rather than processing, akin to how Allocate2D allows initialization, which isn't in processing either, but doesn't allow a default value.

Reply:
👍